plenum/server/replica.py

```diff
@@ -472,9 +476,9 @@ def on_propagate_primary_done(self):
             self._checkpointer.update_watermark_from_3pc()
             if self.isPrimary and (self.last_ordered_3pc[0] == self.viewNo):
                 self.lastPrePrepareSeqNo = self.last_ordered_3pc[1]
-        elif not self.isPrimary:
-            self._checkpointer.set_watermarks(low_watermark=0,
-                                              high_watermark=sys.maxsize)
+        # elif not self.isPrimary:
```

Comment:
I suspect we need to leave this code, since backup instances can not correctly restore last ordered after catchup and have to relax the watermarks to restore last ordered from upcoming 3PC messages.

Reply:
ok